card/spec/lib/card/query_spec.rb

```diff
@@ -302,7 +305,10 @@
 
     it "should sort by count" do
       Card::Auth.as_bot do
-        w = Card::Query.new( :name=>[:in,'*always','*never','*edited'], :sort=>{ :right=>'*follow', :item=>'referred_to', :return=>'count' } )
+        w = Card::Query.new(
+          :name=>[ :in, '*always', '*never', '*edited' ],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside square brackets detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.